src/styles.py

```diff
@@ -11,7 +11,19 @@
 from graphs import file_io, ui, utilities
 from graphs.canvas import Canvas
 
-from matplotlib import pyplot
+from lxml import etree
+
+from matplotlib import pyplot, rc_context
+from matplotlib.figure import Figure
+
+import numpy
+
+import svgutils
+
+
+PREVIEW_XDATA = numpy.linspace(0, 10, 1000)
+PREVIEW_YDATA1 = numpy.sin(PREVIEW_XDATA)
+PREVIEW_YDATA2 = numpy.cos(PREVIEW_XDATA)
```

Comment:
I think sin/cos make sense. They are recognizable and show clear features. 

Perhaps fewer data points would be good so that styles that use markers instead of just lines actually have visible markers. Not sure if that would still translate perfectly with sin/cos though.

Reply:
It's probably fine for now in this PR. But perhaps later on we can tweak this equation a bit so that it will look nice with few data points, so that it will work when showing markers.